wear/src/main/java/io/homeassistant/companion/android/home/views/DetailsPanelView.kt

```diff
@@ -0,0 +1,163 @@
+                                    asset = CommunityMaterial.Icon.cmd_brightness_7,
+                                    colorFilter = ColorFilter.tint(Color.White)
+                                )
+                            },
+                            modifier = Modifier.padding(bottom = 8.dp)
+                        )
+                    }
+                }
+
+                // Color temp
+                val supportsColorTemp = supportedColorModes?.contains(SUPPORT_COLOR_TEMP) ?: false
+                val supportsColorTempComplete =
+                    supportsColorTemp || ((attributes["supported_features"] as Int) and SUPPORT_COLOR_TEMP_DEPR == SUPPORT_COLOR_TEMP_DEPR)
+                if (supportsColorTempComplete && attributes["color_mode"] == SUPPORT_COLOR_TEMP) {
+                    val minValue = (attributes["min_mireds"] as? Number)?.toFloat() ?: 0f
+                    val maxValue = (attributes["max_mireds"] as? Number)?.toFloat() ?: 0f
+                    var currentValue = (attributes["color_temp"] as? Number)?.toFloat() ?: 0f
+                    if (currentValue < minValue)
+                        currentValue = minValue
+                    if (currentValue > maxValue)
+                        currentValue = maxValue
+
+                    item {
+                        Row(
+                            modifier = Modifier.fillMaxWidth().padding(horizontal = 8.dp)
+                        ) {
+                            Text(stringResource(R.string.color_temp))
+                        }
+                    }
+                    item {
+                        InlineSlider(
+                            value = currentValue,
+                            onValueChange = onColorTempChanged,
+                            steps = 20,
+                            valueRange = minValue..maxValue,
+                            decreaseIcon = {
+                                Image(
+                                    asset = CommunityMaterial.Icon3.cmd_thermometer_minus,
+                                    colorFilter = ColorFilter.tint(Color.White)
+                                )
+                            },
+                            increaseIcon = {
+                                Image(
+                                    asset = CommunityMaterial.Icon3.cmd_thermometer_plus,
+                                    colorFilter = ColorFilter.tint(Color.White)
+                                )
+                            },
+                            modifier = Modifier.padding(bottom = 8.dp)
+                        )
+                    }
+                }
+            }
+
+            item {
+                ListHeader(R.string.details)
+            }
+            item {
+                Row(modifier = Modifier.fillMaxWidth().padding(horizontal = 8.dp)) {
+                    Text(stringResource(R.string.entity_id) + ": ${entity.entityId}")
+                }
+            }
```

Comment:
Is the entity ID useful information to have on your watch? I don't think so, but otherwise it should be at the bottom of the list, not at the top.

Reply:
Yeah not sure indeed. Figured I just show the default Entity parameters. Will happily move it to the bottom though